sources/packages/backend/apps/db-migrations/src/sql/SFASPartTimeApplications/Add-col-py-maximum-support.sql

```diff
@@ -0,0 +1,39 @@
+-- Add column cspt_award for SFAS Part Time Application.
+ALTER TABLE 
+  sims.sfas_part_time_applications 
+ADD 
+  COLUMN IF NOT EXISTS cspt_award NUMERIC(10);
+
+COMMENT ON COLUMN sims.sfas_part_time_applications.cspt_award IS 'CSPT Award Amount (sail_extract_data.sail_cspt_amt).';
+
+-- Add column csgd_award for SFAS Part Time Application.
+ALTER TABLE 
+  sims.sfas_part_time_applications 
+ADD 
+  COLUMN IF NOT EXISTS csgd_award NUMERIC(10);
```

Comment:
I see that the existing columns(awards and startdate,enddate) are nullable in DB. Are they suppose to be nullable @andrewsignori-aot ??

![image](https://github.com/bcgov/SIMS/assets/54600590/d5d7e155-ec27-451b-a4ec-c822e03b2dd9)

Reply:
I do not have a clear answer to this. Are they supposed to come as "zero' if the grant is not present in the SFAS/SAIL disbursement?
In case we do not have an answer and the documentation does not state anything I would recommend following what we already have in place for the existing ones. In case we need to change we can change all together. Does it make sense?